### DISCUSSION TITLE
Welcome to SEI-120G Discussions!

### DISCUSSION THREAD

**Discussion by binarykore** [binarykore/SEI120G#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by TestSubject1221** [comment]
does it work to black the firmware for white universal? cuz the firmware version of black is different from this white one

**Reply by binarykore** [reply]
Nope.. And yes, its a Different Config Binary or Firmware..

**Comment by mon888** [comment]
What's the new login and password for the LT90T_1.01.1_v03 iteration?

**Reply by binarykore** [reply]
Paid Credentials..

**Comment by ebchamath** [comment]
what are the networking loging details By usung Port 4719 Or 8357 (telnet or SSH)?

**Reply by binarykore** [reply]
No idea yet.. Try SPI and Pin2Pwn..
